### PR TITLE
ember-cli-babel@6 compatibility

### PR DESCRIPTION
**Pull request by tylerturdenpants**

### CODE REVIEW

File: app/helpers/await.js [hunk 1]
```diff
@@ -1 +1 @@
-export { default, await } from 'ember-promise-helpers/helpers/await';
+export { default, awaitFor } from 'ember-promise-helpers/helpers/await';
```
Comment (fivetanley) [inline app/helpers/await.js]: As far as I can tell, this export doesn't actually exist?

Comment (fivetanley) [inline app/helpers/await.js]: Ah, this was the autogenerated thing from Ember CLI and you were updating it so it wasn't using `async` (which i guess is reserved?). Can you change this line to `export { default } from 'ember-promise-helpers/helpers/await';`?

**Comment by fivetanley** [comment]
Left a comment but otherwise looks great. Thanks for your help!

**Comment by tylerturdenpants** [comment]
We are 🍏 !

**Comment by jrjohnson** [comment]
@fivetanley ember-cli-babel bumped up to v6 last weekend so this issue is now super visible and - if possible - a merge and release would be greatly appreciated.  Thanks.

**Comment by fivetanley** [comment]
@jrjohnson @tylerturdenpants 1.0.3 was just released. Thanks so much!!

**Pull request merged**